packages/cli-repl/src/run.ts

```diff
@@ -220,6 +221,7 @@ async function main() {
     });
   } catch (e: any) {
     console.error(`${e?.name}: ${e?.message}`);
+    console.error('full error: ' + util.inspect(e, { depth: null }));
```

Comment:
Leftover? (Fwiw, I do this a ton too…)

Reply:
yeah. There's nothing more there anyway ;)

Reply:
Would be nice if we could access the most recent error from the test somehow.